ironfish-cli/src/commands/service/ceremony.ts

```diff
@@ -29,6 +29,14 @@ export default class Ceremony extends IronfishCommand {
       description: 'S3 bucket to download and upload params to',
       default: 'ironfish-contributions',
     }),
+    downloadPrefix: Flags.string({
+      char: 'b',
+      parse: (input: string) => Promise.resolve(input.trim()),
+      required: false,
+      description: 'Prefix for contribution download URLs',
+      // TODO: update this to non-dev endpoint to avoid rate limiting
+      default: 'https://pub-6a239e04e140459087cf392ffc3245b1.r2.dev',
```

Comment:
This isn't the bucket, right? I assume it's like an account name

Reply:
its the dev testing url. We will replace it with ironfish.network domain name when close to launch.

Reply:
Yea that is basically a link to the bucket